packages/apps/sale/src/screens/Cart/ActiveCartScreen.tsx

```diff
@@ -0,0 +1,205 @@
+    if (activeCart) {
+      headerActionsProvider.registerModel('sale_active_cart', {
+        model: 'com.axelor.apps.sale.db.Cart',
+        modelId: activeCart?.id,
+        disableMailMessages: !mobileSettings?.isTrackerMessageEnabled,
+        attachedFileScreenTitle: activeCart?.name,
+        actions: [
+          {
+            customComponent: (
+              <DoubleIcon
+                topIconConfig={{
+                  name: 'plus-lg',
+                  color: Colors.primaryColor.background,
+                  size: 13,
+                }}
+                bottomIconConfig={{
+                  name: 'basket2-fill',
+                  color: Colors.secondaryColor_dark.background,
+                }}
+                predefinedPosition={'bottom-right'}
+                topIconPosition={{bottom: -7, right: -7}}
+              />
+            ),
+            iconName: 'basket2-fill',
+            key: 'setAside',
+            order: 10,
+            title: I18n.t('Sale_SetAside'),
+            onPress: () => {},
+          },
+          {
+            customComponent: (
+              <DoubleIcon
+                topIconConfig={{
+                  name: 'trash3-fill',
+                  color: Colors.errorColor.background,
+                  size: 13,
+                }}
+                bottomIconConfig={{
+                  name: 'basket2-fill',
+                  color: Colors.secondaryColor_dark.background,
+                }}
+                predefinedPosition={'bottom-right'}
+                topIconPosition={{bottom: -7, right: -7}}
+              />
+            ),
+            iconName: 'basket2-fill',
+            key: 'emptyCart',
+            order: 20,
+            title: I18n.t('Sale_EmptyCart'),
+            onPress: () => {},
+          },
+        ],
+      });
+    }
+  }, [
+    Colors.errorColor.background,
+    Colors.primaryColor.background,
+    Colors.secondaryColor_dark.background,
+    I18n,
+    activeCart,
+    mobileSettings,
+  ]);
+
+  const handleChangeCustomer = useCallback(
+    newCustomer => {
+      setSelectedCustomer(newCustomer);
+      if (newCustomer != null) {
+        dispatch(
+          (updateCart as any)({partnerId: newCustomer?.id, cart: activeCart}),
+        );
+      }
+    },
+    [activeCart, dispatch],
+  );
+
+  const sliceFunctionData = useMemo(
+    () => ({cartId: activeCart?.id}),
+    [activeCart?.id],
+  );
+
+  if (activeCart == null) {
+    return <Label type="info" message={I18n.t('Sale_NoCartAvailable')} />;
+  }
+
+  return (
+    <Screen
+      removeSpaceOnTop
+      fixedItems={
+        <Button
+          iconName="check-lg"
+          title={I18n.t('Sale_ValidateCart')}
+          onPress={() => {}}
+        />
+      }>
+      <SearchListView
+        actionList={[
+          {
+            iconName: 'plus-lg',
+            title: I18n.t('Sale_AddProduct'),
+            onPress: () => {},
+          },
+        ]}
+        isHideableSearch={true}
```

Comment:
You can leave the SearchBar on the CartLines, it can be useful even through it was not on the screenshot 😁